sample-apps/radiology/lib/infers/segmentation.py

```diff
@@ -56,8 +56,8 @@ def __init__(
     def pre_transforms(self, data=None) -> Sequence[Callable]:
         return [
             LoadImaged(keys="image", reader="ITKReader"),
-            EnsureTyped(keys="image", device=data.get("device") if data else None),
```

Comment:
this can be first.. to push the basic image into GPU.. and the move channel can happen inside the GPU

Reply:
Agree